pkg/config/defaults.go

```diff
@@ -22,7 +22,7 @@ const (
 	// Please keep updated with each new release
 	Version = "0.38.5"
 	// DefaultDAAddress is the default address for the data availability layer
-	DefaultDAAddress = "http://localhost:26658"
+	DefaultDAAddress = "http://localhost:7980"
 	// DefaultSequencerAddress is the default address for the sequencer middleware
 	DefaultSequencerAddress = "localhost:50051"
 	// DefaultSequencerRollupID is the default rollup ID for the sequencer middleware
```

Comment:
we can remove these i think

Reply:
Yeah, I guess we can just fill up DefaultNodeConfig with those